main.go

```diff
@@ -96,7 +96,7 @@ func execBin() error {
 	}
 
 	// run migrations
-	prepareCmd := exec.Command(cmd, "prepare", "db")
+	prepareCmd := exec.Command(cmd, "prepare", "db", "--no-prompt")
```

Comment:
I chose to use the flag here because otherwise I'd have to pipe two subprocesses which would likely make this more flakey in the future. 

Anyone running this script after merge will either be updating an existing setup, or a brand new 1.0.2; the zip file is always pointing at the latest version, so we should be good.